### ISSUE TITLE
Speedtest wrong time

### ISSUE THREAD

**Issue by xawoger** [mimugmail/opn-repo#237]
I have timezone set to europe/london but speedtest in it's logs is showing time one hour before. So when it's 16:00 at my location it shows 15:00. Any way to resolve this issue?

**Comment by jaydenthorup** [comment]
Same issue here 9:05 AM is showing as 15:05 but only when I run them manually. Does this mean that the cron jobs are triggered in gmt, or the gui is wrong?

**Comment by jaydenthorup** [comment]
I am running every hour to see if cron is running at the wrong time, or the reporting is skewed with the cron job.

**Comment by jaydenthorup** [comment]
GUI seems to be correct. Cron seems to be correct. The Gui has a GMT label and upon personal testing the time is running correct. Make sure to verify the difference between GMT and your local time zone.

**Comment by SiLenCe-WB** [comment]
Yep, times are only displayed in GMT format but, in my case, I need CEST time. Having an option to chose timezone would be nice. Or simply just stick to the time settings in opnsense.
